ContactList/ContactList/SearchResultController.swift

```diff
@@ -0,0 +1,56 @@
+//
+//  SearchResultController.swift
+//  ContactList
+//
+//  Created by Maksym Humeniuk on 5/15/19.
+//  Copyright © 2019 Maksym Humeniuk. All rights reserved.
+//
+
+import UIKit
+
+class SearchResultController: UITableViewController {
+    
+    @IBOutlet weak var searchResultLabel: UILabel!
+    
+    var mainTableView: TableViewController!
+    var personsArray: [Person]!
+    var filteredPersons = [Person]()
+    var resultStringsArray = [NSAttributedString]()
+    var searchText = ""
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        self.personsArray = Search.getPersonsArrayFromDictionary(from: mainTableView.groupedPersons)
+    }
+
+    // MARK: - Table view data source
+    override func numberOfSections(in tableView: UITableView) -> Int {
```

Comment:
If you calculate nothing here, you don't need to override this method.
Default implementation return 1.